dreamplace/params.json

```diff
@@ -219,5 +219,9 @@
 "deterministic_flag" : {
     "descripton" : "whether require run-to-run determinism, may have efficiency overhead", 
     "default" : 0
+    },
+"use_dp_for_circuit_training" : {
```

Comment:
As mentioned, try to switch a name if you mean use DREAMPlace for circuit training.

Reply:
Fixed.